### PR TITLE
44 modify the last 2 sections on hone page with responsive perspective to fit tablet and phone view

### PR DESCRIPTION
**Pull request by jasonzhwang**
implement responsive of last two parts of the other section page.

(Please comment header, hero section and the first part of the other sections first to check the responsive,  otherwise,
it will affect the behaviour while resizing the window)

https://github.com/taodemy/taotalk/assets/25715518/f2656e2b-f8a8-4cf9-8689-c1953c74e60c

### CODE REVIEW

File: src/components/Home/OtherSections/index.tsx [hunk 1]
```diff
@@ -29,19 +29,25 @@ const OtherSections = () => {
...
+      <section className="bg-tk_greyLight">
+        <div className="mx-auto grid sm:px-2 sm:py-4 md:px-8 md:py-10 lg:max-w-[1180px] lg:grid-cols-2 lg:px-0 lg:pb-0 lg:pt-[26px] xl:gap-[26px] xl:px-0 xl:pb-0">
+          <div className="mx-0 flex flex-col sm:ml-0 sm:py-4 md:py-[60px] lg:my-auto lg:ml-[100px] lg:block">
+            <HeadingDescription
+              headingContent="Increase your vocabulary"
+              descriptionContent="Traditional and new effective approaches to word study"
+            />
+            <Button outline={false} color="cyanLight" label="Textbook →" />
+          </div>
+          <img src="/vocabulary.svg" alt="Vocabulary" className="mx-0 my-auto sm:w-full" />
```
Comment (Double-Twelve) [inline src/components/Home/OtherSections/index.tsx:41]: should be `mx-auto my-auto sm:h-[200px] md:h-[363px] lg:h-auto`

File: src/components/Home/OtherSections/HeadingDescription/index.tsx [hunk 2]
```diff
@@ -7,9 +7,13 @@ interface HeadingDescriptionProps {
 
 const HeadingDescription = ({ headingContent, descriptionContent }: HeadingDescriptionProps) => {
   return (
-    <div className="headingDescriptionContainer">
-      <h2 className="heading--h2">{headingContent}</h2>
-      <p className="paragraph--p1">{descriptionContent}</p>
+    <div className="sm:flex sm:flex-col">
```
Comment (Double-Twelve) [inline src/components/Home/OtherSections/HeadingDescription/index.tsx:10]: is it necessary to add `sm` ? just write `flex flex-col` directly

File: src/components/Home/OtherSections/HeadingDescription/index.tsx [hunk 3]
```diff
@@ -7,9 +7,13 @@ interface HeadingDescriptionProps {
...
+      <h2 className="lg:w-9/10 sm:w-full sm:text-center sm:text-[32px] md:text-[48px] lg:w-[474px] lg:text-left">
+        {headingContent}
+      </h2>
+      <p className="p1 py-[24px] sm:w-full sm:text-center sm:text-[12px] md:text-[18px] lg:w-[397px] lg:pb-[34px] lg:pt-[30px] lg:text-left">
+        {descriptionContent}
+      </p>
```
Comment (Double-Twelve) [inline src/components/Home/OtherSections/HeadingDescription/index.tsx:16]: why need to add `sm:w-full` ?

File: src/components/Home/OtherSections/index.tsx (hunk 1 above)
Comment (jasonzhwang) [reply src/components/Home/OtherSections/index.tsx]: Hi Shawn,

The issues that you have mentioned have been fixed:
1. remove small media query for mobile as unnecessary
2. add missing height for mobile tablet and desktop by using h-[] or h-auto to control the image size
3. remove xl as there is no large desktop mode in figma design

Cheers

File: src/components/Home/OtherSections/index.tsx [hunk 4]
```diff
@@ -6,7 +6,7 @@ import styles from './OtherSections.module.scss';
 const OtherSections = () => {
   return (
     <>
-      <section className={styles.languageSectionContainer}>
+      {/* <section className={styles.languageSectionContainer}>
```
Comment (tatoMa) [inline src/components/Home/OtherSections/index.tsx:9]: Will this comment going to be used in the future?

**Review by Eric-ZheranLi — APPROVED** [review]
Looks good to me

**Pull request merged**